rgw/admin/user_test.go

```diff
@@ -0,0 +1,115 @@
+package admin
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	fakeUserResponse = []byte(`
+{
+  "tenant": "",
+  "user_id": "dashboard-admin",
+  "display_name": "dashboard-admin",
+  "email": "",
+  "suspended": 0,
+  "max_buckets": 1000,
+  "subusers": [],
+  "keys": [
+    {
+      "user": "dashboard-admin",
+      "access_key": "4WD1FGM5PXKLC97YC0SZ",
+      "secret_key": "YSaT5bEcJTjBJCDG5yvr2NhGQ9xzoTIg8B1gQHa3"
+    }
+  ],
+  "swift_keys": [],
+  "caps": [],
+  "op_mask": "read, write, delete",
+  "system": "true",
+  "admin": "false",
+  "default_placement": "",
+  "default_storage_class": "",
+  "placement_tags": [],
+  "bucket_quota": {
+    "enabled": false,
+    "check_on_raw": false,
+    "max_size": -1,
+    "max_size_kb": 0,
+    "max_objects": -1
+  },
+  "user_quota": {
+    "enabled": false,
+    "check_on_raw": false,
+    "max_size": -1,
+    "max_size_kb": 0,
+    "max_objects": -1
+  },
+  "temp_url_keys": [],
+  "type": "rgw",
+  "mfa_ids": []
+}`)
+)
+
+func TestUnmarshal(t *testing.T) {
+	u := &User{}
+	err := json.Unmarshal(fakeUserResponse, &u)
+	assert.NoError(t, err)
+}
+
+func (suite *RadosGWTestSuite) TestUser() {
+	suite.SetupConnection()
+	co, err := New(suite.endpoint, suite.accessKey, suite.secretKey, nil)
+	co.Debug = true
+	assert.NoError(suite.T(), err)
+
+	suite.T().Run("fail to create user since no UID provided", func(t *testing.T) {
+		_, err = co.CreateUser(context.TODO(), User{Email: "leseb@leseb.io"})
```

Comment:
```suggestion
		_, err = co.CreateUser(context.TODO(), User{Email: "leseb@example.com"})
```
And for others of this same pattern. We don't want to be the source of email/domain harvesting, even if this domain isn't real *today* . :-)